src/modules/mc_rate_control/mc_rate_control_params.c

```diff
@@ -413,3 +413,62 @@ PARAM_DEFINE_INT32(MC_BAT_SCALE_EN, 0);
  */
 PARAM_DEFINE_FLOAT(MC_DTERM_CUTOFF, 0.f);
 
+/**
+ * Inertia matrix, XX component
+ *
+ * @unit kg.m^2
+ * @decimal 5
+ * @increment 0.00001
+ * @group Multicopter Attitude Control
```

Comment:
The group changed in the refactor. I'm also wondering if it's worth hiding these somewhere else to not scare anyone off initially.

Reply:
I set it to Multicopter Rate Control initially. What else do you suggest? "Dynamic model"?

Reply:
I'm not sure, just wondering about perceived complexity for existing users. We don't have a great tuning gui, so many end up manipulating the parameters directly. Keeping that group reasonably simple/accessible might be a feature.

Reply:
"Vehicle model" may be good. Also for when we roll land-detector into a more generic vehicle model module with mass, etc.

Reply:
I like it.
Should I also rename the PARAMETER (VM_INERTIA_*) as it is not limited to multirotors?

Reply:
done